index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.sortItems();
+    this.currencyLength();
+  }
 
-  get(pos) {}
+  sortItems() {
+    this.items.sort((a,b)=>a - b);
+  }
 
-  max() {}
+  currencyLength() {
+    this.length = this.items.length;
+  }
 
-  min() {}
+  get(pos) {
+    if(this.items[pos]) {
+      return pos;
+    }
+    throw new Error('OutOfBounds');
+  }
 
-  sum() {}
+  zeroError() {
+    if(this.length === 0) {
+      throw new Error('EmptySortedList');
+    }
+  }
 
-  avg() {}
+  max() {
+    this.zeroError();
+    return this.items[this.length - 1];
+  }
+
+  min() {
+    this.zeroError();
+    return this.items[0];
+  }
+
+  sum() {
+    return this.items.reduce((acc, number)=>{
```

Comment:
Boa! Aqui daria pra ter feito o reduce em uma linha somente. E por se tratar de uma soma dos elementos, vc nem precisa passar o valor inicial do acumulador! Assim ele assume como valor inicial o primeiro elemento do array ;-)

return this.items.reduce((sum, num) => sum + num);